### ISSUE TITLE
[Accessibility: ACC-271.1] Input field: No Clear visible focus

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12617]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
There should be strong and clear four-sided visible focus provided for all the UI elements.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Platform Table Search](https://sap.github.io/fundamental-ngx/#/platform/table/basic)
For the ‘Search field’ there is no visible focus provided but only cursor is available when the UI receives focus.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/9f875fc4-6355-49ec-bb0d-4d09e38b3ae8">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**
